src/main/java/com/autotune/database/table/lm/KruizeLMExperimentEntry.java

```diff
@@ -0,0 +1,168 @@
+/*******************************************************************************
+ * Copyright (c) 2023 Red Hat, IBM Corporation and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+package com.autotune.database.table.lm;
+
+import com.autotune.analyzer.utils.AnalyzerConstants;
+import com.autotune.database.helper.GenerateExperimentID;
+import com.fasterxml.jackson.databind.JsonNode;
+import jakarta.persistence.*;
+import org.hibernate.annotations.JdbcTypeCode;
+import org.hibernate.type.SqlTypes;
+
+/**
+ * This is a Java class named KruizeExperimentEntry annotated with JPA annotations.
```

Comment:
Update the java class, table name & fields

Reply:
done